isotope/service/pkg/srv/prometheus/handler.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this currentFile except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus
+
+import (
```

Comment:
curious if you would want to support 
https://github.com/istio/fortio#server-urls-and-features
by reusing the code the the fortio/fhttp/http_{server|utils}.go ?

Reply:
For this project, some of those parameters are set by a "script" at start time. I don't think I want to support those query string parameters.

Reply:
I see you have a SleepCommand command and can change payload but still I wonder if it'd be useful to support the probability mode and the other features and reuse the code (in the next version, this is fine as is/for a start)